packages/tasks_panel/src/extension.ts

```diff
@@ -20,19 +18,19 @@ export async function activate(context: ExtensionContext): Promise<void> {
   extensionPath = context.extensionPath;
   const outputChannel = window.createOutputChannel(TASKS_EXPLORER_ID);
   initializeLogger(context, outputChannel);
-  const logger = getLogger();
-  const swa = new SWATracker(
-    "SAPSE",
-    "vscode-tasks-explorer-tasks-panel",
-    // We ignore error code `204` because it appears in every user interaction.
-    (err: string | number) => {
-      if (err !== 204) {
-        logger.error(err.toString());
-      }
-    }
-  );
-
-  initSWA(swa);
+  // const logger = getLogger();
```

Comment:
do we want to leave these comments?

Reply:
I leave it as a reference, when we will take a new analytics library - easier to integrate.